setup.py

```diff
@@ -139,48 +244,28 @@ def initialize_options(self) -> None:
           #
           # https://github.com/aws/aws-cli/blob/develop/setup.py
           'PyYAML<5.3',
-          # sqlalchemy-vertica-python 0.5.5 introduced
-          # https://github.com/bluelabsio/sqlalchemy-vertica-python/pull/7
-          # which fixed a bug pulling schema information from Vertica
-          'sqlalchemy-vertica-python>=0.5.5,<0.6',
-          # sqlalchemy-redshift 0.7.7 introduced support for Parquet
-          # in UNLOAD
-          'sqlalchemy-redshift>=0.7.7',
-          # This is currently vendored in
-          # records_mover/db/postgres/sqlalchemy_postgres_copy.py but
-          # once this PR is merged and a new version published, we can
-          # use the new upstream version:
-          #
-          # https://github.com/jmcarp/sqlalchemy-postgres-copy/pull/14
-          #
-          # 'sqlalchemy-postgres-copy>=0.5,<0.6',
-          'pybigquery',
-          # https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/issues/195
-          #
-          # sqlalchemy 1.3.16 seems to have (accidentally?) introduced
-          # a breaking change that affects sqlalchemy-redshift:
-          #
-          # https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/issues/195
-          'sqlalchemy!=1.3.16',
           # Not sure how/if interface will change in db-facts, so
           # let's be conservative about what we're specifying for now.
           'db-facts>=3,<4',
-          'odictliteral',
-          # we rely on exception types from smart_open,
-          # which seem to change in feature releases
-          # without a major version bump
-          'smart_open>=1.8.4,<1.9.0',
           'chardet',
-          's3-concat>=0.1.7,<0.2'
           'tenacity>=6<7'
       ],
       extras_require={
+          'airflow': airflow_dependencies,
+          'db': db_dependencies,
           'gsheets': gsheet_dependencies,
-          'movercli': gsheet_dependencies + ['typing_inspect',
-                                             'docstring_parser',
-                                             'psycopg2-binary',
-                                             'pandas<2',
-                                             'pyarrow'],
+          # TODO: or should this be cli + cli-minimal?
+          'cli': cli_dependencies_base,
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker